### DISCUSSION TITLE
BSC_project_IPFS2023 - Week1: The growth of spike in winter wheat

### DISCUSSION THREAD

**Discussion by giscus** [Illustratien/BSC_project_IPFS2023#2] [General]
# BSC_project_IPFS2023 - Week1: The growth of spike in winter wheat

http://localhost:7931/posts/Week1/

**Comment by Illustratien** [comment]
Student can reply their `code` here for discussion!

**Comment by Illustratien** [comment]
Question 2 discussion goes here
